lib/server/index.js

```diff
@@ -52,6 +52,25 @@ class GameServer {
     process.stdout.write(`Connection from ${socket.conn.remoteAddress}\n`);
   }
 
+  listGames() {
```

Comment:
At least return an array and rename the method to something like `getCurrentlyRunningGames`.